TWLight/static/css/new-local.css

```
@@ -189,6 +191,298 @@ LOGIN PARTIAL CSS
+    }
+    .partner-glider-description.truncate-overflow{
+        --max-lines: 4;
+        max-height: calc(var(--lh) * var(--max-lines));
+        overflow: hidden;
+    }
+    .partner-glider-description.truncate-overflow:after {
+        content: "";
+        text-align: right;
+        position: absolute;
+        bottom: 0;
+        right: 0;
+        width: 40%;
+        height: 1.2em;
+        background: linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1) 50%);
+    }
+}
+/* For tablets */
+@media only screen
+and (min-device-width : 768px)
+and (max-device-width : 1024px) {
+    .more-item{
+        color: #3366CC;
+        font-weight: 400;
+    }
+    .more-item:hover{
+        color: #3366CC;
+        text-decoration: underline;
+    }
+    .more-item.active{
+        color: #3366CC;
+        font-weight: 600;
+        text-decoration: underline;
+    }
+    .more-option{
+        border-style: solid;
+        border-color: #3366CC;
+        border-width: thin;
+    }
+    .partner-glider-button{
+        width: 98%;
+        color: #FFFFFF;
+        background-color: #3366CC;
+    }
+    .partner-glider-button:hover{
+        color: #FFFFFF;
+        background-color: #3366CC;
+        font-weight: 500;
+    }
+    .partner-image-description-container{
+        position: relative;
+        width: 458px;
+        height: 150px;
+        margin-left: 30px;
+        margin-right: auto;
+    }
+    .partner-glider-image{
+        padding-right: 42px;
+        max-height: 105px;
+        max-width: 100%;
+    }
+    .partner-glider-description{
+        position: absolute;
+        top: 40px;
+        right: 29.5px;
+        white-space: normal;
+    }
+    .partner-glider-description.truncate-overflow{
+        --max-lines: 4;
+        max-height: calc(var(--lh) * var(--max-lines));
+        overflow: hidden;
+    }
+    .partner-glider-description.truncate-overflow:after {
+        content: "";
+        text-align: right;
+        position: absolute;
+        bottom: 0;
+        right: 0;
+        width: 25%;
+        height: 1.2em;
+        background: linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1) 50%);
+    }
+}
+/* For screens larger than tablets */
+.more-item{
+    color: #3366CC;
+    font-weight: 400;
+}
+.more-item:hover{
+    color: #3366CC;
+    text-decoration: underline;
+}
+.more-item.active{
+    color: #3366CC;
+    font-weight: 600;
+    text-decoration: underline;
+}
+.more-option{
+    border-style: solid;
+    border-color: #3366CC;
+    border-width: thin;
+}
+.partner-glider-button{
+    width: 98%;
+    color: #FFFFFF;
+    background-color: #3366CC;
+}
+.partner-glider-button:hover{
+    color: #FFFFFF;
+    background-color: #3366CC;
+    font-weight: 500;
+}
+.partner-image-description-container{
+    position: relative;
+    height: 176px;
+    margin-left: 25px;
+    margin-right: 30px;
+}
+.partner-glider-image{
+    max-width: 100%;
+    height: 125px;
```

Must have overlooked this in my image scaling work. This change resolves the only remaining partner logo stretching.
```suggestion
    max-height: 125px;
```

---

Done!